Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -180,7 +190,19 @@ const routes = [
             inviteKey: route.query.inviteKey,
             inviteEmail: route.query.email,
         }),
-        meta: { validStates: [UserState.notRegistered] },
+        meta: {
+            validStates: [UserState.notRegistered],
+        },
+    },
+    {
+        path: CONFIRM_TERMS_OF_SERVICE_PATH,
+        component: ConfirmTermsOfServiceView,
+        meta: {
+            validStates: [
+                UserState.confirmTermsOfService,
+                UserState.registered,
+            ],
+        },
     },
```

Comment:
Shouldn't the only valid state be UserState.confirmTermsOfService?

Reply:
Agreed.